azure-spring-boot-samples/azure-active-directory-spring-boot-sample/src/main/resources/static/App/Scripts/app.js

```diff
@@ -1,27 +1,32 @@
 ﻿'use strict';
-angular.module('todoApp', ['ngRoute', 'AdalAngular'])
-    .config(['$routeProvider', '$httpProvider', 'adalAuthenticationServiceProvider', function ($routeProvider, $httpProvider, adalProvider) {
+angular.module('todoApp', ['ngRoute', 'MsalAngular'])
+    .config(['$routeProvider', '$httpProvider', 'msalAuthenticationServiceProvider', function ($routeProvider, $httpProvider, msalProvider) {
 
         $routeProvider.when("/Home", {
             controller: "homeCtrl",
             templateUrl: "/App/Views/Home.html",
         }).when("/TodoList", {
             controller: "todoListCtrl",
             templateUrl: "/App/Views/TodoList.html",
-            requireADLogin: true,
+            requireLogin: true,
         }).when("/UserData", {
             controller: "userDataCtrl",
             templateUrl: "/App/Views/UserData.html",
         }).otherwise({redirectTo: "/Home"});
 
-        adalProvider.init(
+        window.applicationConfig = {
+            clientID: 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx'
+        };
+
+        msalProvider.init(
             {
-                instance: 'https://login.microsoftonline.com/',
-                tenant: 'xxxorg.onmicrosoft.com',
-                clientId: 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx',
-                extraQueryParameter: 'nux=1',
+                authority: 'https://login.microsoftonline.com/xxxorg.onmicrosoft.com',
+                clientID: applicationConfig.clientID,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['applicationConfig' is not defined.](https://app.codacy.com/manual/Microsoft/azure-spring-boot/pullRequest?prid=4946864)